canonical-websites/www.ubuntu.com
# Update urls for /download/iot and break to more detailed pages

## Done

- Update urls for /download/iot and break to more detailed pages
- Created new copy docs
- Updated redirects
- Updated the nav
- Moved second OS install into new page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - [/download/iot](http://0.0.0.0:8001/download/iot) with the [copy doc](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit#)
  - [/download/iot/raspberry-pi-2-3](http://0.0.0.0:8001/download/iot/raspberry-pi-2-3 ) the [copy doc](https://docs.google.com/document/d/1Ig906jbgB39QHw61uHMvIsRbSBUjchJBDkMZl0WzlWM/edit#)
  - [/download/iot/kvm ](http://0.0.0.0:8001/download/iot/kvm ) the [copy doc](https://docs.google.com/document/d/1FHIArLpuAyLe2gnDwymefvzc4Lg8dSGvN8QAl8wX18Q/edit)
   - [/download/iot/qualcomm-dragonboard-410c](http://0.0.0.0:8001/download/iot/qualcomm-dragonboard-410c ) the [copy doc](https://docs.google.com/document/d/1nV3782gOD7q0RxWDUmQXJoGG_paRtIpFpa_eBbMwdJI/edit)
  - [/download/iot/orange-pi-zero](http://0.0.0.0:8001/download/iot/orange-pi-zero ) the [copy doc](https://docs.google.com/document/d/1vMQ9DesjpV3fA1iVpKxdHYANbf2Y-Mp5rIXGnV3HypY/edit#heading=h.8dmoxibe4jyh)
  - [/download/iot/raspberry-pi-compute-module-3](http://0.0.0.0:8001/download/iot/raspberry-pi-compute-module-3 ) the [copy doc](https://docs.google.com/document/d/1BO8UI7BNDbyLhLcRSbxzhhaAj-dO6AbYPJUJ0A36vCQ/edit)
  - [/download/iot/intel-joule](http://0.0.0.0:8001/download/iot/intel-joule ) the [copy doc](https://docs.google.com/document/d/1DeFj9aeZoGHm3tlTKGFnvdo3OgJwdHen5AI5hYe7geQ/edit#)
  - [/download/iot/intel-joule-desktop ](http://0.0.0.0:8001/download/iot/intel-joule-desktop ) the [copy doc](https://docs.google.com/document/d/1uniGWQZokDNQIzdUEeq6WGs_x2jxa-vwA03TZ39kCWc/edit#)
  - [/download/iot/up-squared-iot-grove-server](http://0.0.0.0:8001/download/iot/up-squared-iot-grove-server ) the [copy doc](https://docs.google.com/document/d/1-q8NzhmqYvPCAT2AwSv7N9sr_TgJdCc8_gLT7RmaoNE/edit)
   - [/download/iot/intel-nuc ](http://0.0.0.0:8001/download/iot/intel-nuc ) the [copy doc](https://docs.google.com/document/d/1e2aWvVEEXvxDWSbxDzs-_tqKbKbW9sapjCjIRiuAdMg/edit#)
  - [/download/iot/intel-nuc-desktop](http://0.0.0.0:8001/download/iot/intel-nuc-desktop ) the [copy doc](https://docs.google.com/document/d/1hDkrBqpJXWaENYZMSxd8P-XLFQ2LQqS1wNe4o2MGLho/edit#)
  - [/download/iot/samsung-artik-5-10](http://0.0.0.0:8001/download/iot/samsung-artik-5-10 ) the [copy doc](https://docs.google.com/document/d/1PpFCRx06gny9WQ6IT7KrWj23AfCulobxx2pQPkxOQrE/edit#)
  - [/download/iot/samsung-artik-5-10-server](http://0.0.0.0:8001/download/iot/samsung-artik-5-10-server) with the [copy doc](https://docs.google.com/document/d/1YZKKfrEk_RGqXQnfgGJ6dTs8hjcD6Tz88gEFwhWIiJk/edit#)

## Issue / Card

Fixes #2968
